File: KQXDrawer/KQXDrawer/MainTableViewCell.m
//
//  MainTableViewCell.m
//  KQXDrawer
//
//  Created by Kxx.xxQ 一生相伴 on 15/12/4.
//  Copyright © 2015年 asahi_kuang. All rights reserved.
//

#import "MainTableViewCell.h"

@implementation MainTableViewCell

- (void)awakeFromNib {
    // Initialization code
    [self.icon.layer setMasksToBounds:YES];
    [self.icon.layer setCornerRadius:40.f];
    [self.cellLb setFont:BOLD(25.f)];
    [self.cellLb setTextColor:RGBA(21, 106, 30, 1)];
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: KXXDrawer_Swift/KXXDrawer_Swift/MainTableViewCell.swift
//
//  MainTableViewCell.swift
//  KXXDrawer_Swift
//
//  Created by Kxx.xxQ 一生相伴 on 15/12/5.
//  Copyright © 2015年 asahi_kuang. All rights reserved.
//

import UIKit

class MainTableViewCell: UITableViewCell {

    @IBOutlet var icon: UIImageView!
    @IBOutlet var titleLb: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
        self.titleLb.textColor = UIColor(red: 115/255.0, green: 155/255.0, blue: 6/255.0, alpha: 1)
        self.titleLb.font = UIFont.boldSystemFontOfSize(20)
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
